visma/gui/window.py

```diff
@@ -40,6 +40,12 @@ def __init__(self):
         font.setPointSize(12)
         self.setFont(font)
 
+        appIcon = QtGui.QIcon()
+        appIcon.addFile(os.path.abspath('visma/icons/16x16.png'))
```

Comment:
Nice visual touch. Can you move the icons into a folder named **assets**.
```suggestion
        appIcon.addFile(os.path.abspath('assets/icons/16x16.png'))
```

Reply:
Sure thing! Moved icons folder to assets folder :)